Firestore/src/V1/FirestoreClient.php

```diff
@@ -35,6 +35,16 @@
  */
 class FirestoreClient extends FirestoreGapicClient
 {
+    /**
+     * @see FirestoreClient::partitionQueryPaginated
+     * @return \Google\Cloud\Firestore\V1\PartitionQueryResponse
+     * @deprecated use partitionQueryPaginated instead
+     */
+    // public function partitionQuery(array $optionalArgs = [])
+    // {
+    //     return $this->partitionQueryPaginated($optionalArgs)->getPage()->getResponseObject();
+    // }
```

Comment:
This will be uncommented manually in the next Firestore OwlBot PR, as uncommenting it now would break the tests.